tests/params/basic.yml

```diff
@@ -834,6 +885,8 @@ partnership:
 
 prep:
   target: 0.1
+  init: 0.7
```

Comment:
why have the init so much higher than prob?

Reply:
Same as above, but this one was to thread the needle between one race going on and the other not.